.github/ISSUE_TEMPLATE/DISCUSSION.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 💬 Discussion
+about: Create a discussion topic.
+labels: 'note: to discuss'
```

Comment:
Can you update the label by `type: discussion ✍️`

`note: to discuss` is just a **note** for some issues/PR not a **type**.

Reply:
Fixed @lauriejim